src/sentry/api/helpers/group_index/delete.py

```diff
@@ -49,14 +48,6 @@ def delete_group_list(
     # Tell seer to delete grouping records for these groups
     call_delete_seer_grouping_records_by_hash(group_ids)
 
-    # We do not want to delete split hashes as they are necessary for keeping groups... split.
-    GroupHash.objects.filter(
-        project_id=project.id, group__id__in=group_ids, state=GroupHash.State.SPLIT
-    ).update(group=None)
-    GroupHash.objects.filter(project_id=project.id, group__id__in=group_ids).exclude(
-        state=GroupHash.State.SPLIT
-    ).delete()
```

Comment:
I was about to say that we still need this overall delete, just not the `exclude`, but actually deleting the group [deletes the `GroupHash` records](https://github.com/getsentry/sentry/blob/afe8517a0b2438558f08d926656b408f0dc72c51/tests/sentry/tasks/deletion/test_groups.py#L63), so I think getting rid of this entire statement is fine.